docs/reference/5-command-line-tools/1-goliothctl/goliothctl_stream_query.md

```diff
@@ -8,7 +8,7 @@ hide_title: true
 ---
 ## goliothctl stream query
 
-Run a query against all data stored in a LightDB Stream in the supplied path.
+Run a query against all data stored in a LightDB stream in the supplied path.
```

Comment:
@miguel91it I would tend to agree, however these docs are generated from the `goliothctl` commands directly, so updates should be made there rather than just patching up the generated docs IMO.

Reply:
Thanks @szczys! I would lean towards keeping this aligned with the latest release content. Do you anticipate a new CLI release soon?

Reply:
I haven't been involved in CLI releases, this may be a question for @miguel91it